### PR TITLE
PUD-1258 - Last known addresses list

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: cypress/integration/notInCustody.feature [hunk 1]
```diff
@@ -10,6 +10,7 @@ Feature: Not in custody
     And Maria confirms the person has a last known address
     And Maria looks up an address by postcode
     And Maria types an address
+    And Maria can see the addresses listed
     And Maria submits the date and email of the recall request received from probation service
```
Comment (philsegal) [inline cypress/integration/notInCustody.feature:14]: Should there be a step to verify the addresses on the CYA/RI page?

Comment (jonnywyatt) [reply cypress/integration/notInCustody.feature:14]: there is one already

Comment (jonnywyatt) [reply cypress/integration/notInCustody.feature:14]: https://github.com/ministryofjustice/manage-recalls-e2e-tests/blob/main/cypress/integration/stepDefinitions/book.js#L223-L228

Comment (philsegal) [reply cypress/integration/notInCustody.feature:14]: Has it been updated to validate addresses?

Comment (jonnywyatt) [reply cypress/integration/notInCustody.feature:14]: We don't do validation in E2E tests, that's covered by integration tests

**Review by philsegal — APPROVED** [review]
Approving to unblock. Unsure we are verifying the addresses on CYA that this change is adding.

**Pull request merged**